src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -57,7 +57,7 @@ export const KarmaScore = ({score}: KarmaScoreProps) => {
                   lineHeight: normalize(24),
                   marginBottom: normalize(8)
                 }}>
-                Your Karma Score
+                What is my Karma Score?
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the text from "Your Karma Score" to "What is my Karma Score?" seems more informative and user-friendly. It gives a clear indication that the following text will explain what the Karma score is.
```diff
- Your Karma Score
+ What is my Karma Score?
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->